src/Components/Login.jsx

```diff
@@ -0,0 +1,71 @@
+import React, { Component } from 'react'
+import { signInUser } from '../state/actions/reduxTokenAuthConfig'
+import { connect } from 'react-redux'
+import LoginForm from './LoginForm'
+
+class Login extends Component {
+  state = {
+    email: '',
+    password: '',
+    errorMessage: ''
+  }
+
+  inputChangeHandler = (e) => {
+    this.setState({
+      [e.target.name]: e.target.value
+    })
+  }
+
+  handleLogin = () => {
+    const { signInUser } = this.props;
+    const { email, password } = this.state;
+      signInUser({ email, password })
+        .then(
+          console.log('Wander on..')
```

Comment:
I don't understand why you are using `console.log` here?

Reply:
The .then requires to have something within it, and it's a good verification for us that signInUser-function was used.

Reply:
Usually this could be the place where you can redirect. If I see correctly you have the login on a separate route, so you can use the react router to navigate back to the index page for example by doing:
```
this.props.history.push('/')
```

Reply:
Cool! We'll do it!